index.d.ts

```diff
@@ -53,8 +53,8 @@ export interface RendererProps<T> {
   children?: React.ReactNode;
 }
 
-export type InjectedRendererProps = Omit<RendererProps<T>, 'iconsClassNameMap'>;
```

Comment:
This is a type alias not an interface, so I would like to avoid the use of interfaces for it and stick with type.

Can we change it to

```
export type InjectedRendererProps<T> = Omit<RendererProps<T>, 'iconsClassNameMap'>;
export type CustomRendererProps<T> = Omit<RendererProps<T>, 'style'>;
```

Reply:
yeah, I think so. But I assumed, that an interface is just a special case of `type` object, is it not? it does not differ from type when it comes to using it in your code

Reply:
they sort o represent the same thing, but semantics for Union types and representing simple type aliases look strange and are not very intuitive when using interaces